R/utilities-simulation.R

```diff
@@ -112,7 +112,7 @@ resetSimulationCache <- function() {
 #' removeSimulationFromCache(sim1) # returns TRUE
 #' removeSimulationFromCache(sim2) # returns FALSE
 removeSimulationFromCache <- function(simulation) {
-  validateIsOfType(simulation, "Simulation")
+  validateIsOfType(simulation, Simulation)

```

Comment:
example of use. We can use either way. Just a convenience to be able to not rely on strings

Reply:
Good idea. Let's merge it.